### PR TITLE
EOS-14816 : Node disk insertion alerts created on SSPL fresh boot

### PR DESCRIPTION
**Pull request by satish-seagate**
Signed-off-by: Satish Darade <satish.darade@seagate.com>

## Problem Statement
<pre>
  <code>
    Story Ref (if any):
    Please add Problem statement here...
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Please add Problem decription here...
  </code>
</pre>
## Solution
<pre>
  <code>
    Please add summary of the change,List any dependencies that are required for this change.
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ ] Yes
    - [ ] No
  </code>
</pre>
## Unit/Manual Testing Description
<pre>
  <code>
    Please describe the tests that you ran to verify your changes.
  </code>
</pre>

### CODE REVIEW

**Review by sandeepsgt — COMMENTED** [review]
LGTM

File: low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view [hunk 1]
```diff
@@ -1006,6 +1074,55 @@ class SSPLHealthView(object):
...
+        else:
+            return bytearray(value).rstrip(bytearray((0,))).decode('utf-8')
+
+    def _is_physical_drive(self, interfaces_and_property):
+        """If wwn starts with 0x5 then it is physical drive"""
+        return interfaces_and_property["WWN"].startswith("0x5")
+
+    def _is_local_drive(self, interfaces_and_property):
+        return "org.freedesktop.UDisks2.Drive.Ata" in interfaces_and_property and \
+            str(interfaces_and_property["org.freedesktop.UDisks2.Drive"]["Revision"]) != "G265"
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view:1108]: We need support for G280 also now. so check needs to be if not  G265 & G280, then consider as local drive. Can you update in systemd watchdog sensor also

**Review by cdeshmukh — COMMENTED** [review]
Small comment to consider G280 fw version also

File: low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view (hunk 1 above)
Comment (satish-seagate) [reply low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view]: Done

**Review by sandeepsgt — APPROVED** [review]

**Review by thavanathan — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view [hunk 2]
```diff
@@ -1006,6 +1074,55 @@ class SSPLHealthView(object):
...
+            value is None:
+            return ''
+        elif isinstance(value, str):
+            return value
+        elif isinstance(value, bytes):
+            return value.decode('utf-8')
+        else:
+            return bytearray(value).rstrip(bytearray((0,))).decode('utf-8')
+
+    def _is_physical_drive(self, interfaces_and_property):
```
Comment (indrajitzagade) [inline low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view:1102]: Where is this used?

Comment (satish-seagate) [reply low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view:1102]: We haven't used this in the above code. but we may need when we need physical/jbod disk information in health view. like we have in the main code. It is just 2,3 lines of code.

Comment (indrajitzagade) [reply low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view:1102]: ok

**Review by indrajitzagade — APPROVED** [review]

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**